CHANGELOG.md

```diff
@@ -24,6 +24,31 @@ New modules
 Other minor additions
 ---------------------
 
+* Added new operations to `Data.Fin.Subset`:
+  ```
+  _─_ : Op₂ (Subset n)
+  _-_ : Subset n → Fin n → Subset n
+  ```
+
+* Added new proofs to `Data.Fin.Subset.Properties`:
+  ```
+  s⊂s             : p ⊂ q → s ∷ p ⊂ s ∷ q
+  ∣p∣≤∣x∷p∣       : ∣ p ∣ ≤ ∣ x ∷ p ∣
+  
+  p─q⊆p           : p ─ q ⊆ p
+  ∣p─q∣≤∣p∣       : ∣ p ─ q ∣ ≤ ∣ p ∣
+  p∩q≢∅⇒p─q⊂p     : Nonempty (p ∩ q) → p ─ q ⊂ p
+  p∩q≢∅⇒∣p─q∣<∣p∣ : Nonempty (p ∩ q) → ∣ p ─ q ∣ < ∣ p ∣
+  p─q─r≡p─q∪r     : (p ─ q) ─ r ≡ p ─ (q ∪ r)
+  p─q─r≡p─r─q     : (p ─ q) ─ r ≡ (p ─ r) ─ q
```

Comment:
You're just missing `p ─ ⊥ ≡ p` and you'll have that `flip _─_`
is a commutative monoid morphism from `(_∪_, ⊥)` to `(_∘_, id)`.

Reply:
Added :+1: